pkg/server/plugin/upstreamauthority/vault/vault_fake_test.go

```diff
@@ -92,8 +93,7 @@ approle_auth {
   "auth": {
     "client_token": "cf95f87d-f95b-47ff-b1f5-ba7bff850425",
     "policies": [
-      "web",
-      "stage"
+      "test"
```

Comment:
I don't think that this is explicitly consumed and validated in the test. Does this make any difference?

Reply:
You're right, the changes will not affect the results of the test.
I'll revert this.

Reply:
40b9f38